storybook/src/admin-date-time/DatePicker.tsx

```diff
@@ -24,7 +26,8 @@ const Story = () => {
                         <Card>
                             <CardContent>
                                 <Field name="dateOne" label="Date" fullWidth component={FinalFormDatePicker} />
-                                <Field name="dateTwo" label="Clearable" fullWidth clearable component={FinalFormDatePicker} />
+                                <Field name="dateTwo" label="Required" fullWidth required component={FinalFormDatePicker} />
+                                <Field name="dateThree" label="Clearable" fullWidth component={FinalFormDatePicker} />
```

Comment:
I don't think `dateThree` is needed, as the non-required date above is already clearable now.